.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: Check code format
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: install clang-format
+        run: |
+          sudo apt update
+          sudo apt install -y clang-format
+      - name: checkout repository content
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v3
+      - name: run python Script
+        run: |
+          chmod +x ./script/run-clang-format.py
```

Comment:
**suggestion:** We should commit this mod in git and remove this line. What do you think?

Reply:
done with command ` git update-index --chmod=+x ./script/run-clang-format.py  `